rest_api/v2/components/index.yml

```
@@ -3047,8 +3047,8 @@ parameters:
     required: true
     schema:
       type: string
-      enum: ['fr', 'es', 'it', 'en', 'de']
-    description: Language in which the categories are going to be displayed
+      enum: ['fr', 'es', 'it', 'en', 'de', 'pt-BR']
```

Can you put them in MAJ here, so it's ISO ? :)
```suggestion
      enum: ['FR', 'ES', 'IT', 'EN', 'DE', 'PT-BR']
```